x/epochs/module.go

```diff
@@ -136,6 +136,13 @@ func (am AppModule) LegacyQuerierHandler(legacyQuerierCdc *codec.LegacyAmino) sd
 // module-specific GRPC queries.
 func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
+
+	migrator := keeper.NewMigrator(am.keeper)
+
+	// register v1 -> v2 migration
+	if err := cfg.RegisterMigration(types.ModuleName, 1, migrator.Migrate1to2); err != nil {
```

Comment:
this needs to be added to the upgrade logic on `app.go` too (set the UpgradeHandler)

Reply:
Added an UpgradeHandler stub in https://github.com/tharsis/evmos/pull/611